vxtelegram/telegram.py

```diff
@@ -232,6 +245,14 @@ def translate_inbound_message(self, message):
     def handle_outbound_message(self, message):
         # TODO: handle direct replies
         message_id = message['message_id']
+
+        # Handle replies to inline queries separately from text messages
+        if 'type' in message['transport_metadata']:
```

Comment:
Here you can also just do `if message['transport_metadata'].get('type') == 'inline':`.